aws/src/main/java/org/apache/iceberg/aws/RESTSigV4AuthSession.java

```diff
@@ -56,6 +55,7 @@ public class RESTSigV4AuthSession implements AuthSession {
   static final String EMPTY_BODY_SHA256 =
       "e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855";
   static final String RELOCATED_HEADER_PREFIX = "Original-";
+  private static final String X_AMZ_CONTENT_SHA256 = "x-amz-content-sha256";
```

Comment:
make it public and utilize this one the test case above.
```suggestion
 static final String X_AMZ_CONTENT_SHA256 = "x-amz-content-sha256";
```